### PR TITLE
ODC-7698: Replace `CopyPipelineRunButton` with PF5 design

### PR DESCRIPTION
**Pull request by logonoff**
**Fixes**: 

https://issues.redhat.com/browse/ODC-7698

**Solution Description**: 

Remove `react-copy-to-clipboard` dep, remove `CopyPipelineRunButton`, and switch from a monaco-based design to the PatternFly-provided multiline design 

I considered other options, such as rewriting the `CopyPipelineRunButton` using the PatternFly component; however, the presence of the `textId` prop in PF's `ClipboardCopyButton` highly implies that the intended design should accompany a node in the DOM with an id containing the text to be copied. However, the `EditorField` does not have an ID containing just the text, and copying from its parent would include line numbers. 

**Screen shots / Gifs for design review**: 

demo

https://github.com/user-attachments/assets/881f641c-6222-486b-a948-e13be739c9a3

**Unit test coverage report**: 

n/a

**Test setup:**

n/a

**Browser conformance**: 

- [x] Chrome
- [x] Firefox
- [ ] Safari

### CODE REVIEW

File: frontend/packages/pipelines-plugin/src/components/repository/sections/RepositoryOverview.tsx [hunk 1]
```diff
@@ -132,7 +116,12 @@ const RepositoryOverview = () => {
               </Trans>
             </Text>
             <Text>
-              <ClipboardCopy isReadOnly hoverTip="Copy" clickTip="Copied" style={{ flex: '1' }}>
```
Comment (logonoff) [inline frontend/packages/pipelines-plugin/src/components/repository/sections/RepositoryOverview.tsx:135]: also replaced non-i18n strings with their internationalized counterpart

**Review by vikram-raj — COMMENTED** [review]
/lgtm

**Pull request merged**